homeassistant/components/ai_speaker/sensor.py

```diff
@@ -0,0 +1,77 @@
+"""Support for AI-Speaker sensors."""
+from datetime import timedelta
+import logging
+
+from aisapi.ws import AisWebService
+
+from homeassistant.helpers import aiohttp_client
+from homeassistant.helpers.entity import Entity
+
+from .const import DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+SCAN_INTERVAL = timedelta(minutes=60)
+
+
+async def async_setup_entry(hass, config_entry, async_add_entities):
+    """Perform the setup for AI-Speaker status sensor."""
+    _LOGGER.debug("AI-Speaker sensor, async_setup_entry")
+    async_add_entities([AisSensor(hass, config_entry.data)], True)
+
+
+async def async_unload_entry(hass, entry):
+    """Clean up when integrations are removed - add code if you need to do more."""
+    _LOGGER.debug("AI-Speaker sensor, async_unload_entry")
+
+
+class AisSensor(Entity):
+    """AiSpeakerSensor representation."""
+
+    def __init__(self, hass, config_entry_data):
```

Comment:
Don't pass in `hass`. It will be set on the entity when the entity has been added to home assistant. This means you can access `hass` with `self.hass`, after the entity has been added to home assistant.